crates/re_space_view_spatial/src/parts/points3d.rs

```diff
@@ -219,3 +219,23 @@ impl ViewPartSystem for Points3DPart {
         self
     }
 }
+
+/// Run 4 things in parallel
+fn join4<A: Send, B: Send, C: Send, D: Send>(
```

Comment:
I wonder where we should put helpers like these. Yet another helper crate (`re_parallel`)?